test/unittests/mongoBackend/mongoUpdateContextGeo_test.cpp

```diff
@@ -207,6 +207,7 @@ TEST(mongoUpdateContextGeoRequest, newEntityLocAttribute)
     EXPECT_FALSE(a2.hasField("creDate"));
     EXPECT_FALSE(a2.hasField("modDate"));
     EXPECT_STREQ("A1", ent.getObjectField("location").getStringField("attrName"));
+    ASSERT_TRUE(ent.hasField("location"));
```

Comment:
What I mean is tha using that  ASSERT_TRUE test execution is safer. For example, if the "location" field is missing and we don't use ASSERT_TRUE then next line `EXPECT_EQ(2, coordX(ent))` will end in segfault, given that `coordX()` function assumes that the field exists in the entity.